vmc/resource_vmc_cluster.go

```diff
@@ -293,6 +298,9 @@ func resourceClusterDelete(d *schema.ResourceData, m interface{}) error {
 		if err != nil {
 			return resource.NonRetryableError(fmt.Errorf("error while deleting SDDC %s: %v", sddcID, err))
 		}
+		if *task.Status == "FAILED" {
```

Comment:
can FAILED and FINISHED (line 304) be combined into one if with ||? I don't see a difference in the way they are both handled.